assisted_deployment.sh

```diff
@@ -7,18 +7,181 @@ source common.sh
 source assisted_deployment_utils.sh
 
 
+ASSISTED_INSTALL_TEST_INFRA=${ASSISTED_INSTALL_TEST_INFRA:-}
+ASSISTED_INSTALLER_NAMESPACE="${ASSISTED_INSTALLER_NAMESPACE:-assisted-installer}"
+ASSISTED_OPERATOR_INDEX="${ASSISTED_OPERATOR_INDEX:-quay.io/ocpmetal/assisted-service-index:latest}"
+
+
+function deploy_local_storage() {
+  oc adm new-project openshift-local-storage || true
+
+  oc annotate project openshift-local-storage openshift.io/node-selector=''
+
+  cat <<EOF | oc apply -f -
+apiVersion: operators.coreos.com/v1alpha2
+kind: OperatorGroup
+metadata:
+  name: local-operator-group
+  namespace: openshift-local-storage
+spec:
+  targetNamespaces:
+    - openshift-local-storage
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: local-storage-operator
+  namespace: openshift-local-storage
+spec:
+  channel: "4.7"
```

Comment:
I have to try without setting the `channel`. It doesn't sound right to hardcode this value. I tried using `$OPENSHIFT_VERSION` but it didn't seem to work. Will re-test.